### PR TITLE
draggable widget: actions on drag-start and drag-end

### PR DESCRIPTION
**Pull request by BurningTreeC**
this allows actions on drag-start and drag-end

why: it allows selective/conditional applying of "pointer-events: none" on certain elements. Those elements can be "blocking elements" like iframes. Or droppable widgets that contain other droppable widgets, where one can either apply conditionals on the action-string to prevent undesired actions from one or more  droppable widgets, or apply different classes to the droppable div when another droppable should be triggered... and I can imagine many other things

### CODE REVIEW

**Comment by BurningTreeC** [comment]
Hi @Jermolene - is this considerable when I add the docs?

File: core/modules/widgets/draggable.js [hunk 1]
```diff
@@ -67,6 +69,8 @@ DraggableWidget.prototype.execute = function() {
 	// Pick up our attributes
 	this.draggableTag = this.getAttribute("tag","div");
 	this.draggableClasses = this.getAttribute("class");
+	this.dragStartActions = this.getAttribute("startactions");
```
Comment (Jermolene) [inline core/modules/widgets/draggable.js]: I think we should use "startActions", "endActions" for consistency with e.g. LinkCatcher widget.

**Comment by Jermolene** [comment]
Hi @BurningTreeC 

> Hi @Jermolene - is this considerable when I add the docs?

Yes, looks good.

File: core/modules/widgets/draggable.js (hunk 1 above)
Comment (BurningTreeC) [reply core/modules/widgets/draggable.js]: Ok! :+1:

**Comment by BurningTreeC** [comment]
@Jermolene 

done , it's ready to merge :smiley:

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Pull request merged**